tests/mir-opt/inline/small_blocks_inline_threshold.rs

```diff
@@ -0,0 +1,17 @@
+// EMIT_MIR small_blocks_inline_threshold.main.Inline.after.mir
+
+// Test case generated to detect flipped threshold condition on `check_mir_body`.
+// The decision of whether to inline (or not) a block of code is decided by its 
+// cost of inlining, compared to the block's inline threshold
+// Normally small bonus is given to the inline threshold for small blocks, via
+//  threshold += threshold / 4
+// where inline will only be performed if the cost <= threshold.
+
+// This test checks that the decision to increase the threshold for small blocks
+// has been applied correctly. This test was generated by a fuzzer.
+
+use std::env;
+fn main() {
+  let a : Vec<String> = env::args().collect();
```

Comment:
Calling standard library functions sounds fragile if they are ever changed. I think it would make more sense to create a self-contained test case for this. It could also be a unit test for inline to further narrow it down (see other MIR opt tests for how to do that)